documentation/book/appendix_crds.adoc

```diff
@@ -1613,6 +1613,8 @@ Used in: xref:type-KafkaConnectS2I-{context}[`KafkaConnectS2I`]
 |integer
 |image                     1.2+<.<|The docker image for the pods.
 |string
+|buildResources            1.2+<.<|CPU and memory resources to reserve (limits and requests) for the build.
```

Comment:
I don't quite follow. Why is "limits and requests" in parentheses?

Reply:
It is copypaste from the description of other resources.

Reply:
Well, it doesn't make sense there either! Does "using limits and requests" make sense?

Reply:
What about "CPU and memory resources limits and requests for the ..."?